ddl/ddl.go

```diff
@@ -668,6 +668,17 @@ type RecoverInfo struct {
 	CurAutoRandID int64
 }
 
+// delayForAsyncCommit sleeps `SafeWindow + AllowedClockDrift` before a DDL job finishes.
+// It should be called before any DDL that could break data consistency.
+// This provides a safe window for async commit and 1PC to commit with an old schema.
+func delayForAsyncCommit() {
```

Comment:
Do we need to check 1PC here?

Reply:
1PC works in the same way with async commit.

Reply:
"sleep before DDL finishes to make async commit safe" is not accurate?

Reply:
OK. I just change the log without changing the function name. It looks too verbose to also put 1PC to the name